Quality-Assistance.md

```diff
@@ -0,0 +1,96 @@
+## Quality Assistance
+
+[Quality Assistance at Octopus](https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance)
+
+# Everyone
+
+> See [Everyone](https://github.com/OctopusDeploy/People/blob/master/Everyone.md) and these specific examples below.
+
+## I am an egoless, team-oriented person
+
+- I actively build good relationships with individuals in our product delivery teams, including product managers, product designers, and software engineers.
+- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus as a whole.
+- Even when I disagree with something, I commit to the agreed course of action, work enthusiastically toward the best outcome, and learn from the experience.
+- I recognize and accept that we will always be learning and improving how we work, and my approach allows me to tactfully challenge the status quo or confidently adapt to change.
+
+## I elevate product delivery teams
+
+- Product development teams genuinely want my assistance with their work.
+- I raise the level of coordination, collaboration, and clarity in the teams I'm coaching.
```

Comment:
```suggestion
- I raise the standard of coordination, collaboration, and clarity in the teams I'm coaching.
```
(to clarify that by "raise the level of" we do not necessarily mean "raise the _amount_ of")